width: 0.60 # 二维码板的宽度
err_x: 0.15 # 相机和雷达坐标系的x偏移
err_y: -0.01  # 相机和雷达坐标系的y偏移
err_z: -0.12  # 相机和雷达坐标系的z偏移
samll_box_x: 1.5  # 小搜索框x长度的一半
samll_box_y: 0.2  # 小搜索框y长度的一半
samll_box_z: 0.25 # 小搜索框z长度的一半
big_box_x: 2.0  # 大搜索框x长度的一半
big_box_y: 0.5  # 大搜索框y长度的一半
big_box_z: 0.4  # 大搜索框z长度的一半
RING: 80  # 激光雷达线束